src/frontend/src/stores/flowStore.ts

```diff
@@ -116,6 +120,7 @@ const useFlowStore = create<FlowStoreType>((set, get) => ({
       newFlowPool[nodeId].push(data);
     }
     get().setFlowPool(newFlowPool);
+    get().updateFlowPool(nodeId, data);
```

Comment:
Maybe the lines 122 and 123 are doing the same thing

Reply:
correct. removed now